plugins/item_previews/web_client/views/HierarchyWidget.js

```diff
@@ -7,20 +7,18 @@ import ItemPreviewWidget from './ItemPreviewWidget';
 
 wrap(HierarchyWidget, 'render', function (render) {
     render.call(this);
-
-    // Only on folder views:
     if (this.parentModel.resourceName === 'folder' && this._showItems) {
-        // Add the item-previews-container.
-        var element = $('<div class="g-item-previews-container">');
-        this.$el.append(element);
-
-        // Add the item preview widget into the container.
-        this.itemPreviewView = new ItemPreviewWidget({
-            folderId: this.parentModel.get('_id'),
-            parentView: this,
-            el: element
-        })
-        .render();
+        if (!this.itemPreviewView) {
+            this.itemPreviewView = new ItemPreviewWidget({
```

Comment:
The binding of this sub-view to the DOM can be simplified. Just pass `class: 'g-item-preview-container',` as an argument when instantiating the `ItemPreviewWidget`. The new `this.itemPreviewView` will be created with it's own `<div>`element (that's the default type) of the correct class, making the element creation and call to `.setElement(` here unnecessary.

Then you can just do ` this.$el.append(this.itemPreviewView.$el);`.